src/mission_ui.cpp

```diff
@@ -0,0 +1,160 @@
+#include "game.h"
+#include "player.h"
+#include "output.h"
+#include "input.h"
+#include "mission.h"
+#include "weather.h"
+#include "calendar.h"
+
+#include <vector>
+#include <string>
+
+
+void game::list_missions()
+{
+    WINDOW *w_missions = newwin( FULL_SCREEN_HEIGHT, FULL_SCREEN_WIDTH,
+                                 ( TERMY > FULL_SCREEN_HEIGHT ) ? ( TERMY - FULL_SCREEN_HEIGHT ) / 2 : 0,
+                                 ( TERMX > FULL_SCREEN_WIDTH ) ? ( TERMX - FULL_SCREEN_WIDTH ) / 2 : 0 );
+
+    int tab = 0;
```

Comment:
I agree. Since I don't see any reason not to do that (the diff already involves every line), I'll look into it.